src/Microsoft.ML.AutoML/TrainerExtensions/MultiTrainerExtensions.cs

```diff
@@ -229,4 +229,22 @@ public PipelineNode CreatePipelineNode(IEnumerable<SweepableParam> sweepParams,
                 columnInfo.LabelColumnName, columnInfo.ExampleWeightColumnName);
         }
     }
+
+    internal class ImageClassificationExtension : ITrainerExtension
```

Comment:
you may want to make sure that the CLI still runs / doesn't break if someone has image paths in their column names?

Reply:
The main issue I expect is needing to add the right package references to the CLI. Otherwise the CLI builds, and runs correctly from VS, but the packaged/shipped CLI fails at runtime.

Need to add the new packages used to `mlnet.nupkgproj`. 

Thanks to @eerhardt for his help in debugging the CLI when I was building this.